using-timescaledb/continuous-aggregates.md

```diff
@@ -16,145 +16,201 @@ every refresh. Only new and/or invalidated data will be calculated.  Since this
+       AVG(temperature),
+       MAX(temperature),
+       MIN(temperature)
+FROM conditions
+GROUP BY device, bucket;
 ```
-This creates a continuous view called `device_summary` for the aggregation
-query, as well as all of the infrastructure needed for materializing the data
-and keeping the view up-to-date. A `time_bucket` on the time partitioning column
-of the raw hypertable is required in all continuous aggregate views. The
-[`time_bucket`][time-bucket] function in this case has a `bucket_width` of 1
-hour. See the [`CREATE VIEW (Continuous Aggregate)`][api-continuous-aggs-create]
-section of our documentation for all of the options and requirements for the
-command.
-
-You can also create multiple continuous aggregates on the same raw hypertable. For example, we may
-want to query per-device readings aggregated on a daily basis. To do so, we can create a continuous
-aggregate like so:
+
+A `time_bucket` on the time partitioning column of the underlying
+hypertable is required in all continuous aggregate views. If you do
+not provide one, you will get an error. The `time_bucket` function in
+this case has a bucket width of 1 hour.
+
+You can create multiple continuous aggregates on the same
+hypertable. For example, you can create a continuous aggregate
+`condition_summary_daily` over the data with daily buckets:
 
 ```sql
-CREATE VIEW device_summary
-WITH (timescaledb.continuous) --This flag is what makes the view continuous
-AS
-SELECT
-  time_bucket('1 day', observation_time) as bucket, --time_bucket is required
-  device_id,
-  avg(metric) as metric_avg, --We can use any parallelizable aggregate
-  max(metric)-min(metric) as metric_spread --We can also use expressions on aggregates and constants
-FROM
-  device_readings
-GROUP BY bucket, device_id; --We have to group by the bucket column, but can also add other group-by columns
+CREATE VIEW conditions_summary_daily
+WITH (timescaledb.continuous) AS
+SELECT device,
+       time_bucket('1 day::interval, "time") AS bucket,
+       AVG(temperature),
+       MAX(temperature),
+       MIN(temperature)
+FROM conditions
+GROUP BY device, bucket;
 ```
 
-In general, aggregates which can be [parallelized by PostgreSQL][postgres-parallel-agg]
-are allowed in the view definition, this includes most aggregates distributed
-with PostgreSQL. This means that aggregates with `ORDER BY` clauses or `DISTINCT` clauses cannot appear,
-additionally we require that no `FILTER` clauses appear ([though we will allow them in future
-releases](#future-work)).
+Continuous aggregates are supported for any aggregates that can be
+[parallelized by PostgreSQL][postgres-parallel-agg]. This includes
+most aggregates distributed with PostgreSQL, but in particular `ORDER
+BY` and `DISTINCT` cannot be used with continuous aggregates since
+they are not possible to parallelize by PostgreSQL. In addition,
+`FILTER` cannot currently be used with continuous aggregates.
```

Comment:
This section is a bit weird cause it mixes several things. It talks about the supported aggregates and then other query limitations and then back to filtered aggregates not being support.

Reply:
This I didn't follow. It start with discussing what is supported, and then highlights what is not supported mentioning, in particular, `ORDER BY`, `DISTINCT`, and `FILTER`. Not sure what kind of changes you want to see.